lib/ios-deploy.js

```diff
@@ -28,9 +30,11 @@ class IOSDeploy {
   }
 
   async install (app) {
+    const start = new Date();
     try {
       const bundlePathOnPhone = await this.pushAppBundle(app);
       await this.installApplcation(bundlePathOnPhone);
+      log.info(`Installation is successful after ${new Date() - start}ms`);
```

Comment:
I usually prefer to use process.hrtime for performance meassurement

Reply:
we don't know if the installation is successful, since `this.installApplcation` (btw there is a type in the method name) continues even after a timeout happens

Reply:
That is successful for us. The phones are something flaky and don't notify you back this happens really rarely so I let that slide and call it also successful